tests/unit/cluster-cli.tcl

```diff
@@ -2,22 +2,6 @@
 
```

Comment:
While looking at it, most of the functionality being tested in the file was the cli, the "setup/migrating" stuff was all orchestrated through `redis-cli`. I also vaguely remember we wanted to have a test that actually did cli testing, since that was a gap. So I think there is value in keeping this mainly focused on cli testing.